research/2022-08-20-introducing-waku-product.md

```diff
@@ -0,0 +1,170 @@
+---
+layout: post
+name:  "Waku for All DApps"
+title:  "Waku for All DApps"
+date:   2022-11-01 00:00:00 +0000
+author: franck
+published: true
+permalink: /waku-for-all-dapps
+categories: waku, dapp, platform, operator
+summary: Waku adoption push has shifted from Web to all platforms. 
+image: /img/black-waku-logo-with-name.png
+discuss: TODO
+---
+
+<!--
+  Audience: Reader curious about Waku/following Waku
+  Goal:
+    - Trigger Ha-ha moment for Waku possibilities
+      - Explain shift from Web to all dApps
+    - Make it clear that Waku is being field tested, it is more than a research project
+    - Ideas on how to contribute/participate/play with Waku
+-->
+
+# Waku for the Web
+
+We are building Waku to be the communication layer for Web3:
+a network and suite of protocols to enable secure, censorship-resistant, privacy-preserving communications that can run on any device.
```

Comment:
Could we instead say ' Waku network offers a collection of protocols to chose from for your messaging needs. It enables secure, censorship-resistant, spam-protected communication for its users.'